### ISSUE TITLE
Build failed  TrustedWireITCase.ignoresPkixErrors

### ISSUE THREAD

**Issue by YamStranger** [jcabi/jcabi-http#104]
Results : 
 
Failed tests:  

```
  TrustedWireITCase.ignoresPkixErrors:51 HTTP response status is not equal to 404:
500 Internal Server Error [https://api.travis-ci.org/test]
Status: 500 Internal Server Error
Date: Thu, 23 Jul 2015 09:30:48 GMT
Via: 1.1 vegur
Content-Type: application/json
Connection: close
Server: Cowboy
 
{"error":"Travis encountered an error, sorry :("}
Expected: HTTP response with status 404
     but: was <500 Internal Server Error [https://api.travis-ci.org/test]
Status: 500 Internal Server Error
Date: Thu, 23 Jul 2015 09:30:48 GMT
Via: 1.1 vegur
Content-Type: application/json
Connection: close
Server: Cowboy
 
{"error":"Travis encountered an error, sorry :("}> 
 
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0 
```

**Comment by YamStranger** [comment]
@dmarkov this issue blocks #85, please fix it

**Comment by dmarkov** [comment]
@yegor256 dispatch this issue please

**Comment by yegor256** [comment]
@YamStranger how can I reproduce this error? I mean, do you always see it or only sometimes?

**Comment by YamStranger** [comment]
@yegor256 I see it always, when run tests. I found it when prepare fix (https://github.com/jcabi/jcabi-http/pull/103). I think that it is only on my pc, but not. I push "almost empty" test function, just to test builds on Travis CI and AppVeyor and they fail.

**Comment by YamStranger** [comment]
@yegor256, @dmarkov any updates?

**Comment by YamStranger** [comment]
@yegor256, @dmarkov any updates?

**Comment by yegor256** [comment]
@YamStranger it's fixed in c337006 and f1f5a0e
